### PR TITLE
Feat/transfer transport and helpers

### PR DESCRIPTION
**Pull request by Alex-A4**
Summary:
- Completed `nekoton/helpers` logic
- Added all models for dart-side (just to reduce time later)
- Merge main here

### CODE REVIEW

File: packages/flutter_nekoton_bridge/lib/nekoton/core/models/transaction.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    required List<Message> outMessages,
+  }) = _Transaction;
+
+  factory Transaction.fromJson(Map<String, dynamic> json) =>
+      _$TransactionFromJson(json);
+
+  const Transaction._();
+
+  @override
+  int compareTo(Transaction other) => -createdAt.compareTo(other.createdAt);
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/lib/nekoton/core/models/transaction.dart:31]: Maybe just swap compared values and remove minus?

File: packages/flutter_nekoton_bridge/lib/nekoton/crypto/constants.dart [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const kSecretKeyLength = 32;
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/lib/nekoton/crypto/constants.dart:1]: k prefixes for constants again.

File: packages/flutter_nekoton_bridge/lib/nekoton/crypto/derived_key/constants.dart [hunk 3]
```diff
@@ -0,0 +1 @@
+const kDerivedKeySignerName = 'DerivedKeySigner';
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/lib/nekoton/crypto/derived_key/constants.dart:1]: And one more prefix.

File: packages/flutter_nekoton_bridge/lib/nekoton/crypto/encrypted_key/constants.dart [hunk 4]
```diff
@@ -0,0 +1 @@
+const kEncryptedKeySignerName = 'EncryptedKeySigner';
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/lib/nekoton/crypto/encrypted_key/constants.dart:1]: Another constant prefix.

File: packages/flutter_nekoton_bridge/lib/nekoton/crypto/ledger_key/constants.dart [hunk 5]
```diff
@@ -0,0 +1 @@
+const kLedgerKeySignerName = 'LedgerKeySigner';
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/lib/nekoton/crypto/ledger_key/constants.dart:1]: And one more const prefix.

File: packages/flutter_nekoton_bridge/lib/nekoton/core/accounts_storage/models/constants.dart [hunk 6]
```diff
@@ -0,0 +1 @@
+const kAccountsStorageKey = '__core__accounts';
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/lib/nekoton/core/accounts_storage/models/constants.dart:1]: -

File: packages/flutter_nekoton_bridge/lib/nekoton/core/models/internal_message.dart [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+part 'internal_message.freezed.dart';
+part 'internal_message.g.dart';
+
+@freezed
+class InternalMessage with _$InternalMessage {
+  const factory InternalMessage({
+    String? source,
+    required String destination,
+    required String amount,
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/lib/nekoton/core/models/internal_message.dart:11]: No, in rust its u64

[the remark this answers is not in the document]
Comment (nesquikm) [reply packages/flutter_nekoton_bridge/lib/nekoton/core/models/internal_message.dart:11]: Maybe this is a perfect place to use BigInt or even Money (but seems like there is no information for creating Money, so BigInt)?

[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/lib/nekoton/core/models/internal_message.dart:11]: I'll put it to the last PR in order, to [tests_pr](https://github.com/broxus/nekoton_bridge/pull/38)

**Review by nesquikm — APPROVED** [review]

**Pull request merged**